README.markdown

```diff
@@ -871,6 +871,7 @@ Nginx may terminate a request early with (at least):
 * 400 (Bad Request)
 * 405 (Not Allowed)
 * 408 (Request Timeout)
+* 413 (Request Entity Too Large)
```

Comment:
This markdown file is automatically generated from `doc/HttpLuaModule.wiki`, BTW. I'll update it on my side. Just let you know :)